src/styles/score.scss

```diff
@@ -6,6 +6,22 @@ $scoreTable-transition-leave-time: 300ms;
   font-size: 1.25rem;
 }
 
+.score-table td:nth-child(3), td:nth-child(4), td:nth-child(5), td:nth-child(6), td:nth-child(7) {
```

Comment:
It's good you know pseudo selectors, but wouldn't your life be easier if for every threshold `div` you add class '.score' and use it here?
BTW this solution requires you to keep it in sync with form structure. (Broken by my suggestion about not rendering `<td style={{ display: 'none' }} />` ;) )

Alternatively you can use bootstrap provided utility classes to containers which content you want to center  `d-flex justify-content-center`
[flex-utilities](https://getbootstrap.com/docs/4.0/utilities/flex/) (check other utils, as you can replace most of your custom styles with bootstrap utils)

Reply:
I'll discover flex goodies in a few days. Now I leave is as a regular table.